molgenis-data-annotators/src/main/java/org/molgenis/data/annotator/tabix/TabixRepository.java

```diff
@@ -122,19 +122,26 @@ public EntityMetaData getEntityMetaData()
 		try
 		{
 			org.molgenis.data.annotator.tabix.TabixReader.Iterator iterator = reader.query(queryString);
-			String line = iterator.next();
-			while (line != null)
+			if (iterator != null)
 			{
-				Entity entity = toEntity(line);
-				if (entity.getLong(positionAttributeName) == pos)
+				String line = iterator.next();
```

Comment:
I did not touch this code, the changes I made were adding the if (iterator != null) { ... } block around existing code.